### DISCUSSION TITLE
S0 Drehstromzähler DSZ15D-3x80A MID

### DISCUSSION THREAD

**Discussion by Sn4k3E4t3r** [huizebruin/s0tool#88] [General] [answered]
do you have an idea how I could manage that I can count my three-phase meter Eltako DSZ15D-3x80A MID the S0 pulses? 

he can also blink in the display but this is not an IR LED or something?

**Comment by huizebruin** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/62996429/208300041-70a0a227-6b1c-4201-ac7e-128de33d30f3.png)

if u open the top u must see the s0 port, there u can connect it to.
Dutch article about the DSZ15D-3x80A MID
[https://www.eltako.com/fileadmin/downloads/vl/_bedienung/DSZ15D-3x80A_28380015-1_flaem.pdf](https://www.eltako.com/fileadmin/downloads/vl/_bedienung/DSZ15D-3x80A_28380015-1_flaem.pdf)
German artikel about it.
[https://www.elektroland24.de/media/pdf/2c/fa/46/DSZ15D-3X80A_Datenblatt%20DSZ15D-3X80A.pdf](https://www.elektroland24.de/media/pdf/2c/fa/46/DSZ15D-3X80A_Datenblatt%20DSZ15D-3X80A.pdf)
German settings artikel about it.
[https://www.elektroland24.de/media/pdf/16/17/f5/DSZ15D-3X80A_Anleitung%20DSZ15D-3X80A.pdf](https://www.elektroland24.de/media/pdf/16/17/f5/DSZ15D-3X80A_Anleitung%20DSZ15D-3X80A.pdf)
English information about is.
[https://www.eltako.com/fileadmin/downloads/en/_datasheets/Datasheet_DSZ15D-3x80A.pdf](https://www.eltako.com/fileadmin/downloads/en/_datasheets/Datasheet_DSZ15D-3x80A.pdf) 

And its working with the s0tool and his code, i have a couple of users that have this one in use with the s0tool and his code.
